doc/manual/manual.tex

```diff
@@ -565,9 +566,15 @@ \subsubsection{Boundary conditions}
 $\Gamma_{0,\mathbf u}\cup\Gamma_{\text{prescribed},\mathbf
   u}\cup\Gamma_{\parallel,\mathbf u}\cup\Gamma_{\text{traction},\mathbf
   u}=\Gamma$ and
-$\Gamma_{D,T}\cup\Gamma_{N,T}=\Gamma$. No boundary conditions have to be
-posed for the compositional fields at those parts of the boundary where flow is either
-tangential to the boundary or points outward.
+$\Gamma_{D,T}\cup\Gamma_{N,T}=\Gamma$. 
+
+Boundary conditions have to be posed for the compositional fields only
```

Comment:
I have not heard the word "posed", only "imposed" in this context. Was this on purpose?

Also, I am of course aware of the the fact that you can/should only describe values on inflow boundaries, but how do we handle this in ASPECT? I don't think we have any specific logic for this in place right now. The only thing I can think of right now is that one can have a prescribed inflow velocity for a complete boundary id and then leave this id off the list of prescribed boundaries. Maybe mention this here? Alternatively, we can open an issue to test this and possibly make an example.